CHANGELOG.md

```diff
@@ -289,7 +295,9 @@ Version 4.0.0 is a backwards-incompatible release. Please see the [upgrade instr
 [v4.2.1]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v4.1.0...v4.2.1
 [v4.3.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v4.2.1...v4.3.0
 [v5.0.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v4.3.0...v5.0.0
+[v5.0.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v5.0.0...v5.0.1
 
+[#330]: https://github.com/forseti-security/terraform-google-forseti/pull/476
```

Comment:
Update to `[#476]` and make sure link is updated.

Reply:
Done